src/ListComponent.js

```diff
@@ -141,11 +136,15 @@ const ListComponent = (props) => {
                                                     onClick={() => deleteTask(item.id)}> 
                                                     Delete 
                                                 </Button> 
-                                                <Button variant = "light"
+                                                <Button style={{marginRight:"10px"}}
+                                                    variant = "light"
                                                     onClick={() => editTask(id)}> 
                                                     Edit 
                                                 </Button> 
-                                              <input type = "checkbox" value={done} onChange={()=>handleChange("done")} /> done
+                                                <Button variant = {item.completed ? "dark" : "light"}
```

Comment:
I changed the checkbox to a button to match the other buttons. We can go back to checkbox if you both want to.